modules/post/windows/gather/credentials/filezilla_server.rb

```diff
@@ -342,37 +360,33 @@ def parse_server(data)
         perm['dirlist']    = opt[6].text rescue "<unknown>"
         perm['dirsubdirs'] = opt[7].text rescue "<unknown>"
         perm['autocreate'] = opt[9].text rescue "<unknown>"
+        perm['host']    = settings['ftp_bindip']
 
-        if opt[8].text == "1"
-          perm['home'] = "true"
-        else
-          perm['home'] = "false"
-        end
-        perms << perm
+        opt[8].text == "1" ? (perm['home'] = "true") : (perm['home'] = "false")
 
+        perms << perm
       end
 
-      user.elements.to_a("//User//IpFilter//Allowed").each do |allowed|
+      user.elements.to_a("IpFilter/Allowed").each do |allowed|
       end
-      user.elements.to_a("//User//IpFilter//Disallowed").each do |disallowed|
+      user.elements.to_a("IpFilter/Disallowed").each do |disallowed|
       end
 
       account['host'] = settings['ftp_bindip']
-      perm['host']    = settings['ftp_bindip']
       account['port'] = settings['ftp_port']
       account['ssl']  = settings['ssl']
       creds << account
 
-      vprint_status("    Collected the following credentials:")
       vprint_status("    Username: %s" % account['user'])
       vprint_status("    Password: %s" % account['password'])
-      vprint_status("       Group: %s" % account['group'])
+      vprint_status("       Group: %s" % account['group']) if account['group'] != nil
       vprint_line("")
```

Comment:
Ruby convention for string interpolation should be used. Also no need for vprint_line(""). Just vprint_line is sufficient.